### ISSUE TITLE
./gradlew build fails in multi-project build

### ISSUE THREAD

**Issue by Kaboomboom3** [wpilibsuite/GradleRIO#292]
When the build task is ran from a multi-project build it fails with a NullPointerException. Both projects use the GradleRIO plugin and the root project depends on the sub-project.

```
$ ./gradlew build
> Task :extractTestJNI FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':extractTestJNI'.
> java.lang.NullPointerException (no error message)
```

An example project can be found here: https://github.com/Kaboomboom3/GradleRIO-Build-Test

**Comment by JacisNonsense** [comment]
This is already fixed in #289 and is pending publish

**Comment by Kaboomboom3** [comment]
Awesome thanks!
